packages/web-components/src/theme/index.ts

```diff
@@ -0,0 +1,2 @@
+export * from './design-tokens';
```

Comment:
I need to add bundle-size / tree-shaking tests. After that we might reconsider the way how we export and consume tokens.